libs/core/src/components/pds-accordion/pds-accordion.scss

```diff
@@ -18,10 +18,10 @@ details {
   --number-animation-transform-timing: 200ms;
 
   --spacing-details-padding-inline: var(--pine-spacing-150);
-  --spacing-details-padding-block: var(--pine-spacing-100);
-  --spacing-summary-padding-block: var(--pine-spacing-50);
+  --spacing-details-padding-block-end: calc(var(--pine-spacing-200) + 2) //18 + 2 = 18px;
```

Comment:
I'd be curious to understand the calculations here vs prev token usage. Not a blocker